### PR TITLE
[expotools] Move ios-shell-app task to expotools

### PR DESCRIPTION
**Pull request by sjchmiela**
# Why

Migrating tools to `expotools` 👍 

# How

Added a new command file to `expotools` — `IosShellApp`. Copied the code from `tools-public` responsible for creating iOS shell apps. To create the options set I had to dive deep into `xdl` and infer args from there.

# Test Plan

I can't test it properly before SDK39 drops — right now kernel expects `expo-updates` to be present in the standalone app and `xdl` thinks the app we're building is still on SDK38 (`exponent-view-template/…/Supporting/EXSDKVersions.plist` renders `38.0.0` as _latest_).

At least the build steps run, see [build](https://github.com/expo/expo/runs/1040914498).

### CODE REVIEW

File: tools/expotools/src/commands/IosShellApp.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { IosShellApp, ImageUtils } from '@expo/xdl';
+import fs from 'fs';
+import path from 'path';
+import sharp from 'sharp';
+
+import * as Directories from '../Directories';
+
+async function resizeIconWithSharpAsync(iconSizePx, iconFilename, destinationIconPath) {
```
Comment (bbarthec) [inline tools/expotools/src/commands/IosShellApp.ts:8]:
```suggestion
async function resizeIconWithSharpAsync(iconSizePx: number, iconFilename: string, destinationIconPath: string) {
```

File: tools/expotools/src/commands/IosShellApp.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  const filename = path.join(destinationIconPath, iconFilename);
+
+  // sharp can't have same input and output filename, so load to buffer then
+  // write to disk after resize is complete
+  let buffer = await sharp(filename).resize(iconSizePx, iconSizePx).toBuffer();
+
+  fs.writeFileSync(filename, buffer);
+}
+
+async function getImageDimensionsWithSharpAsync(dirname, basename) {
```
Comment (bbarthec) [inline tools/expotools/src/commands/IosShellApp.ts:18]:
```suggestion
async function getImageDimensionsWithSharpAsync(dirname: string, basename: string) {
```

**Review by bbarthec — APPROVED** [review]
I'm surprised that we stil have a way not to respect writing TS types everywhere 🤔

**Comment by sjchmiela** [comment]
TypeScript did not complain, so 🤷‍♂️. Thanks for catching that!

**Pull request merged**